d2l-scroll-wrapper.html

```diff
@@ -127,98 +127,84 @@
 				overflow-x: auto;
 				width: 100%;
 				box-sizing: border-box;
-				@apply(--d2l-scroll-wrapper-inner);
+				@apply --d2l-scroll-wrapper-inner;
 			}
 
-			:host-context([dir="rtl"])[h-scrollbar] .wrapper,
+			:host([is-rtl][h-scrollbar]) .wrapper,
 			:host([h-scrollbar]) .wrapper {
-				@apply(--d2l-scroll-wrapper-h-scroll);
+				@apply --d2l-scroll-wrapper-h-scroll;
 			}
 
-			:host-context([dir="rtl"]):host([h-scrollbar]) .wrapper {
-				@apply(--d2l-scroll-wrapper-h-scroll);
-			}
-
-			:host-context([dir="rtl"])[scrollbar-right] .wrapper,
+			:host([is-rtl][scrollbar-right]) .wrapper,
 			:host([scrollbar-left]) .wrapper {
-				@apply(--d2l-scroll-wrapper-left);
-			}
-
-			:host-context([dir="rtl"]):host([scrollbar-right]) .wrapper {
-				@apply(--d2l-scroll-wrapper-left);
+				@apply --d2l-scroll-wrapper-left;
 			}
 
-			:host-context([dir="rtl"])[scrollbar-left] .wrapper,
+			:host([is-rtl][scrollbar-left]) .wrapper,
 			:host([scrollbar-right]) .wrapper {
-				@apply(--d2l-scroll-wrapper-right);
-			}
-
-			:host-context([dir="rtl"]):host([scrollbar-left]) .wrapper {
-				@apply(--d2l-scroll-wrapper-right);
+				@apply --d2l-scroll-wrapper-right;
 			}
 
 			.action {
-				@apply(--d2l-scroll-wrapper-action);
+				@apply --d2l-scroll-wrapper-action;
 			}
 			.sticky {
-				@apply(--d2l-scroll-wrapper-action-hidden);
-				@apply(--d2l-scroll-wrapper-sticky);
+				@apply --d2l-scroll-wrapper-action-hidden;
+				@apply --d2l-scroll-wrapper-sticky;
 			}
 
-			:host-context([dir="rtl"]) .left .action,
+			:host([is-rtl]) .left .action,
 			.right .action {
-				@apply(--d2l-scroll-wrapper-action-end);
+				@apply --d2l-scroll-wrapper-action-end;
 			}
 
-			:host-context([dir="rtl"]) .right .action,
+			:host([is-rtl]) .right .action,
 			.left .action {
-				@apply(--d2l-scroll-wrapper-action-start);
+				@apply --d2l-scroll-wrapper-action-start;
 			}
 
-			:host-context([dir="rtl"]) .left,
+			:host([is-rtl]) .left,
 			.right {
 				float: right;
 			}
 
-			:host-context([dir="rtl"]) .right,
+			:host([is-rtl]) .right,
 			.left {
 				float: left;
 			}
 
 			:host([h-scrollbar]) .sticky {
-				@apply(--d2l-scroll-wrapper-action-visible);
+				@apply --d2l-scroll-wrapper-action-visible;
 			}
 
 			/* Hide the start/end buttons depending on the state */
 			:host([scrollbar-right]) .right,
 			:host([scrollbar-left]) .left {
-				@apply(--d2l-scroll-wrapper-action-hidden);
+				@apply --d2l-scroll-wrapper-action-hidden;
 			}
 		</style>
 		<template is="dom-if" if="{{showActions}}" strip-whitespace>
```

Comment:
Also, `dom-if` in Polymer 2.0 is an optional import, so not sure if that needs to be imported explicitly yet, or only when we convert to a 2.0 element:
https://www.polymer-project.org/2.0/docs/upgrade#optional-features